experimental/packages/opentelemetry-instrumentation-grpc/src/grpc-js/types.ts

```diff
@@ -59,3 +62,10 @@ export type MakeClientConstructorFunction = typeof grpcJs.makeGenericClientConst
 
 export type { HandleCall } from '@grpc/grpc-js/build/src/server-call';
 export type { PackageDefinition } from '@grpc/grpc-js/build/src/make-client';
+
+export type metadataCaptureType = {
+  client: {
+    captureRequestMetadata: (span: Span, metadata: grpcJsTypes.Metadata | grpcTypes.Metadata) => void,
```

Comment:
It seems like the type is been duplicated in `experimental/packages/opentelemetry-instrumentation-grpc/src/grpc/types.ts`.

As the metadata is declared to accept both `grpcJsTypes.Metadata` and `grpcTypes.Metadata`, we can move this type declaration to the package root `experimental/packages/opentelemetry-instrumentation-grpc/src/types.ts`.

Reply:
True it is common between the two, will move it to `src/types.ts`